src/weekly/utils.rs

```diff
@@ -74,27 +75,28 @@ pub(crate) fn weekday_from_string(input: &str) -> Fallible<Weekday> {
 /// assert_eq!(morning.0, 14);
 /// assert_eq!(morning.0, 5);
 /// ```
-pub(crate) fn time_from_string(input: &str) -> Fallible<(u8, u8)> {
+#[context("failed to parse time string {}", input)]
```

Comment:
I guess this will result in duplicated `input` in the error message, as the inner errors are also printing it. It is likely enough to only print it once, either here or at the bottom of the chain (my vote goes to the latter).